src/pdbpp.py

```diff
@@ -131,6 +131,15 @@ def rebind_globals(func, newglobals):
             _newfunc(func.func, newglobals), *func.args, **func.keywords
         )
 
+    # Python 3.11
+    _pdb = __import__("pdb", fromlist=("_ModuleTarget", "_ScriptTargetu"))
```

Comment:
```suggestion
    _pdb = __import__("pdb", fromlist=("_ModuleTarget", "_ScriptTarget"))
```

Possibly? At least it looks fishy